### PR TITLE
Adds a Code of Conduct.

### PR DESCRIPTION
**Pull request by MrsTonedOne**
:cl: MrStonedOne in cooperation with Feminist Frequency, Geek Feminism, and the ADA Initiative 
code: Added a code of conduct to the repo to signify our commitment to weeding out abusive and triggering behaviors in open source spaces.
/:cl:

@KorPhaeron

### CODE REVIEW

**Comment by JJRcop** [comment]
adapted vs somewhat loosely inspired

**Comment by Supermichael777** [comment]
Webscale and Mongo I (sorta) understand but what do you have against python readability standards?

Spacing is a valid option but we are a deeply tabbed repo.

**Comment by Chnkr** [comment]
10/10 shitposting

**Comment by optimumtact** [comment]
this is stolen code, I made this

**Comment by duncathan** [comment]
is stolen code against our code of conduct?

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment include:
+
+* Writing good code
+* Not arguing with maintainers
+
+Examples of unacceptable behavior by participants include:
+
+* Doing things the maintainers don't like due to their personal bias
```
Comment (kevinz000) [inline CODE_OF_CONDUCT.md]: :ok_hand: :clap:

**Comment by jughu** [comment]
Oranges doesnt like anything dark times have arrivered

**Comment by alexkar598** [comment]
But is it okay if a maintainer argues an other maintainer?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+* Expressing support, in any shape or form, for any of the following Triggers: 
+	* Space indentation
+	* Pep 8
+	* Mongo DB
+	* Webscale
```
Comment (PJB3005) [inline CODE_OF_CONDUCT.md]: Then why was tgui ever merged 🤔

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+* Writing good code
+* Not arguing with maintainers
+
+Examples of unacceptable behavior by participants include:
+
+* Doing things the maintainers don't like due to their personal bias
+* Expressing support, in any shape or form, for any of the following Triggers: 
+	* Space indentation
+	* Pep 8
```
Comment (PJB3005) [inline CODE_OF_CONDUCT.md]: *PEP 8

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+Examples of behavior that contributes to creating a positive environment include:
+
+* Writing good code
+* Not arguing with maintainers
+
+Examples of unacceptable behavior by participants include:
+
+* Doing things the maintainers don't like due to their personal bias
+* Expressing support, in any shape or form, for any of the following Triggers:
```
Comment (PJB3005) [inline CODE_OF_CONDUCT.md]: > Doesn't mention Lummox Jr.

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+Project maintainers have the right to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem annoying.
+
+## Scope
+
+This Code of Conduct might apply wherever a maintainer sees you doing bad stuff.
+
+## Enforcement
+
+Instances of abusive, triggering, or otherwise unacceptable behavior may be reported by contacting the project team at #coderbus. We might take you seriously or we might just laugh at you.
```
Comment (PJB3005) [inline CODE_OF_CONDUCT.md]: "our IRC channel, `#coderbus`"

File: CODE_OF_CONDUCT.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+Project maintainers who do not follow or enforce the Code of Conduct in good faith will be fired. Out of a cannon. In to the sun.
```
Comment (PJB3005) [inline CODE_OF_CONDUCT.md]: *into

**Review by PJB3005 — CHANGES_REQUESTED** [review]
Needs more OCOC.

File: CODE_OF_CONDUCT.md (hunk 6 above)
Comment (PJB3005) [reply CODE_OF_CONDUCT.md]: Also, make it say out of an airlock and into the singulo.

**Comment by BeeSting12** [comment]
I can't tell if this is a joke PR or not.

File: CODE_OF_CONDUCT.md (hunk 5 above)
Comment (Cyberboss) [reply CODE_OF_CONDUCT.md]: on Rizon

**Comment by MrStonedOne** [comment]
@Supermichael777 you're triggering me please stop with your micro aggressions.

Spacing is not a valid option because tab width should be up to the reader of the code, not the writer, and requiring they have to use some ide to replace how the code is represented in memory and file to retain that control is stupid when we have a character that was made in the 80s explicitly for the job and allows everybody to see the code exactly as they want without having to change its actual contents.

And pep8 is a programming micro aggression because I still have PTSD from the time that one guy came in to the repo putting up issues reports about how we "violate pep8" treating it with the urgency one might treat a rce exploit.

**Comment by Anonmare** [comment]
Good PR, speedmerge posthaste

**Pull request closed without merging**